CMake/ITKClangFormatSetup.cmake

```diff
@@ -55,7 +55,7 @@ endif()
 if(CLANGFORMAT_EXECUTABLE)
   mark_as_advanced(CLANGFORMAT_EXECUTABLE)
 endif()
-if ( ITK_USE_CLANGFORMAT AND NOT EXISTS "${CLANGFORMAT_EXECUTABLE}")
+if ( ITK_USE_CLANG_FORMAT AND NOT EXISTS "${CLANGFORMAT_EXECUTABLE}")
```

Comment:
`CLANGFORMAT_EXECUTABLE`? Or `CLANG_FORMAT_EXECUTABLE`? 😺

Reply:
We could use `CLANG_FORMAT` consistently.